fastlane/lib/fastlane/setup/setup.rb

```diff
@@ -37,6 +42,10 @@ def is_android?
       Dir["*.gradle"].count > 0
     end
 
+    def is_react_native?
+      SetupIos.new.project_uses_react_native?(path: "./ios")
```

Comment:
This is actually a PR to another branch of an outstanding PR (https://github.com/fastlane/fastlane/pull/8605), we need to merge this one first 👍